app/src/manager/folder.rs

```diff
@@ -80,7 +80,13 @@ impl<'a> Widget for Folder<'a> {
 					self.file_style(v)
 				};
 
-				ListItem::new(format!(" {icon} {}", readable_path(k, &self.folder.cwd))).style(style)
+				let display_path = readable_path(k, &self.folder.cwd);
+				let display_full = if MANAGER.show_symlink && v.is_link {
+					format!(" {icon} {} -> {}", display_path, v.link_to.clone().unwrap_or_default().to_string_lossy())
```

Comment:
Seems there is an unnecessary clone?

Reply:
It will be an error without clone.
<img width="1474" alt="image" src="https://github.com/sxyazi/yazi/assets/20698483/be65d6dd-7464-4349-8dfe-3c607face8ed">

Reply:
I'll fix it.